spyder/app/mainwindow.py

```diff
@@ -1049,11 +1049,12 @@ def create_edit_action(text, tr_text, icon):
         self.register_shortcut(doc_action, "_",
                                "spyder documentation")
 
-        if self.help is not None:
-            tut_action = create_action(self, _("Spyder tutorial"),
-                                       triggered=self.help.show_tutorial)
-        else:
-            tut_action = None
+        spyder_vid = ("https://www.youtube.com/playlist"
+                      "?list=PLPonohdiDqg9epClEcXoAPUiK0pN5eRoc")
+        vid_action = create_action(self, _("Tutorial Videos"),
+                                   icon=ima.icon('DialogHelpButton'),

```

Comment:
This icon repeats the one above, and implies troubleshooting or "rescueing" a user vs. teaching them new things (as tutorial videos are indented to), so it should either be replaced with something more appropriate, like a video icon from QtAwesome, or removed.